rules.ini

```
@@ -75,6 +75,8 @@ idTech7 = \.streamdb$
 KiriKiri = (?:^|/)KAGParserEx\.dll$
 Lime_OR_OpenFL = (?:^|/)lime(?:-legacy)?\.[hn]dll$
 Love2D = (?:^|/)love\.dll$
+Marmalade[] = (?:^|\/)angle\/libEGLMarmalade\.dll
+Marmalade[] = (?:^|\/)angle\/libGLESv2Marmalade\.dll
```

No need to escape slash + you can group those 2
```suggestion
Marmalade = (?:^|/)angle/lib(?:EGL|GLESv2)Marmalade\.dll
```